--- /usr/local/etc/httpd/httpd.conf	2018-09-05 01:59:02.000000000 +0300
+++ /usr/local/etc/httpd/httpd.conf	2018-09-05 02:06:04.000000000 +0300
@@ -49,7 +49,7 @@
 # prevent Apache from glomming onto all bound IP addresses.
 #
 #Listen 12.34.56.78:80
-Listen 8080
+Listen 80

 #
 # Dynamic Shared Object (DSO) Support
@@ -171,14 +171,14 @@
 </IfModule>
 #LoadModule dav_fs_module lib/httpd/modules/mod_dav_fs.so
 #LoadModule dav_lock_module lib/httpd/modules/mod_dav_lock.so
-#LoadModule vhost_alias_module lib/httpd/modules/mod_vhost_alias.so
+LoadModule vhost_alias_module lib/httpd/modules/mod_vhost_alias.so
 #LoadModule negotiation_module lib/httpd/modules/mod_negotiation.so
 LoadModule dir_module lib/httpd/modules/mod_dir.so
 #LoadModule actions_module lib/httpd/modules/mod_actions.so
 #LoadModule speling_module lib/httpd/modules/mod_speling.so
 #LoadModule userdir_module lib/httpd/modules/mod_userdir.so
 LoadModule alias_module lib/httpd/modules/mod_alias.so
-#LoadModule rewrite_module lib/httpd/modules/mod_rewrite.so
+LoadModule rewrite_module lib/httpd/modules/mod_rewrite.so

 <IfModule unixd_module>
 #
@@ -211,7 +211,7 @@
 # e-mailed.  This address appears on some server-generated pages, such
 # as error documents.  e.g. admin@your-domain.com
 #
-ServerAdmin you@example.com
+ServerAdmin ydenyshchenk@magento.com

 #
 # ServerName gives the name and port that the server uses to identify itself.
@@ -220,7 +220,7 @@
 #
 # If your host doesn't have a registered DNS name, enter its IP address here.
 #
-#ServerName www.example.com:8080
+ServerName php.lo

 #
 # Deny access to the entirety of your server's filesystem. You must
@@ -278,9 +278,13 @@
 # is requested.
 #
 <IfModule dir_module>
-    DirectoryIndex index.html
+    DirectoryIndex index.php index.html
 </IfModule>

+<FilesMatch \.php$>
+    SetHandler application/x-httpd-php
+</FilesMatch>
+
 #
 # The following lines prevent .htaccess and .htpasswd files from being
 # viewed by Web clients.
@@ -303,7 +307,7 @@
 # Possible values include: debug, info, notice, warn, error, crit,
 # alert, emerg.
 #
-LogLevel warn
+LogLevel debug

 <IfModule log_config_module>
     #
@@ -503,7 +507,7 @@
 #Include /usr/local/etc/httpd/extra/httpd-info.conf

 # Virtual hosts
-#Include /usr/local/etc/httpd/extra/httpd-vhosts.conf
+Include /usr/local/etc/httpd/extra/httpd-vhosts.conf
 
 # Local access to the Apache HTTP Server Manual
 #Include /usr/local/etc/httpd/extra/httpd-manual.conf
